resources/views/Widgets/Header/LanguageDetectionWidget.twig

```diff
@@ -0,0 +1,32 @@
+{% import "Ceres::Widgets.Helper.TwigBuilder" as Twig %}
+
+{% set customClass              = widget.settings.customClass.mobile            %}
+{% set appearance               = widget.settings.appearance.mobile             %}
+{% set redirect                 = widget.settings.redirect.mobile               %}
+
+{{ Twig.do("set textTranslations = {}") }}
+{{ Twig.do("set buttonTranslations = {}") }}
+{{ Twig.set("languageList", Twig.call("services.webstoreConfigurationRepository.getActiveLanguageList")) }}
+{{ Twig.for("language", "languageList") }}
+    {{ Twig.set("languageISO", "services.localizationRepository.getLanguageCode(language)") }}
+    {{ Twig.do("set textTranslations = textTranslations | merge({(languageISO): trans('Ceres::Template.languageDetectionText', [], languageISO ) })") }}
+    {{ Twig.do("set buttonTranslations = buttonTranslations | merge({(languageISO): trans('Ceres::Template.languageDetectionButton', [], languageISO ) })") }}
+{{ Twig.endfor() }}
+
+<div class="language-detection header-fw widget-{{ appearance }}{% if customClass | length > 0 %} {{ customClass }}{% endif %}">
+    <div class="container-max px-0 pr-lg-3">
+        {% if isPreview %}
+            <div class="align-center">
+                Language Detection
+            </div>
+        {% else %}
```

Comment:
If possible, the preview should look like the production output of the widget. You should be able to use your Vue component in the preview as well. Maybe you need to check inside of your component if you are in the ShopBuilder to display the language picker even if the displayed language already matches the browser language. To do so, you can access `$ceres.isShopBuilder` inside of your Vue components.